src/wasm/wasm-ir-builder.cpp

```diff
@@ -839,7 +839,10 @@ Result<> IRBuilder::makeRefIsNull() {
   return Ok{};
 }
 
-// Result<> IRBuilder::makeRefFunc() {}
+Result<> IRBuilder::makeRefFunc(Name func) {
+  push(builder.makeRefFunc(func, wasm.getFunction(func)->type));
```

Comment:
Consider adding a comment about wasm-ir-builder having a stricter makeRefFunc() than wasm-builder, one that will throw an exception if the function being made reference to is not already added to the module.

Reply:
I'll add a general comment about this to the top-level IRBuilder doc comment because this applies to many functions.